lms/templates/main.html

```diff
@@ -52,50 +46,17 @@
       </script>
   % endif
 
-  <%
-    jsi18n_path = "js/i18n/{language}/djangojs.js".format(language=LANGUAGE_CODE)
-  %>
-
-  <script type="text/javascript" src="${static.url(jsi18n_path)}"></script>
-
-  <link rel="icon" type="image/x-icon" href="${static.url(static.get_value('favicon_path', settings.FAVICON_PATH))}" />
-
   <%static:css group='style-vendor'/>
   <%static:css group='${self.attr.main_css}'/>
 
-  % if not uses_pattern_library:
-    % if disable_courseware_js:
-      <%static:js group='base_vendor'/>
-      <%static:js group='base_application'/>
-    % else:
-      <%static:js group='main_vendor'/>
-      <%static:js group='application'/>
-    % endif
-  % else:
-    ## TODO: Update to only bring in RequireJS
-    ## https://openedx.atlassian.net/browse/FEDX-140
-    <%static:js group='base_vendor'/>
-    <%static:js group='base_application'/>
-  % endif
-
-  <script>
-    window.baseUrl = "${settings.STATIC_URL}";
-    (function (require) {
-      require.config({
-          baseUrl: window.baseUrl
-      });
-    }).call(this, require || RequireJS.require);
-  </script>
-  <script type="text/javascript" src="${static.url("lms/js/require-config.js")}"></script>
-  <%block name="js_overrides">
-    ${render_require_js_path_overrides(settings.REQUIRE_JS_PATH_OVERRIDES)}
-  </%block>
+  <%include file="common_assets.html" />
 
   % if not disable_courseware_js:
     <%static:js group='module-js'/>
   % endif
 
   <%block name="headextra"/>
+  <%block name="head_extra"/>
```

Comment:
Do we need to add a new block in this scenario, or can we just reuse the former?